modules/account-lib/src/coin/cspr/transactionBuilder.ts

```diff
@@ -305,12 +306,35 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
     return session;
   }
 
+  /**
+   * Checks whether the transaction has the owner signature
+   * @param {string} pub - public key of the signer
+   * @returns {boolean} true if the pub key already signed th transaction
+   * @private
+   */
+  private isTransactionSignedByPub(pub: string): boolean {
+    return _.findIndex(this.transaction.casperTx.approvals, approval => {
+      const approvalSigner = approval.signer.slice(2).toUpperCase();
```

Comment:
Better if you can use something like this.
https://github.com/BitGo/BitGoJS/blob/STLX-2076-cspr-fix-addSignature/modules/account-lib/src/coin/cspr/constants.ts#L1

Reply:
Fixed with a new utils method called `removeAlgoPrefixFromHexValue`